home/models.py

```diff
@@ -139,23 +158,35 @@ class Answer(models.Model):
     question = models.ForeignKey(Question, on_delete=models.CASCADE)
     content = models.TextField()
     publish_date = models.DateTimeField(default=timezone.now)
-    likes_count = models.IntegerField(default=0)
-    dislikes_count = models.IntegerField(default=0)
+    likes = models.ManyToManyField(Profile, null=True, related_name='user_answer_likes')
+    dislikes = models.ManyToManyField(Profile, null=True, related_name='user_answer_dislikes')
     is_edited = models.BooleanField(default=False)
-    ordering = ['likes_count']
+    ordering = ['publish_date']
 
     def __str__(self):
         return self.content
 
-    def thumb_up_answer(self):
-        self.likes_count = self.likes_count + 1
-
-    def thumb_down_answer(self):
-        self.dislikes_count = self.dislikes_count + 1
-
     def set_is_edited(self, newVal):
         self.is_edited = newVal
 
+    def update_thumbs(self, profile, thumb_type):
```

Comment:
I would make thumb_type an enum of just split this into two separate methods.

Reply:
I decided to split into 2 methods